### PR TITLE
REL: release notes v1.5.6

### PR DESCRIPTION
**Pull request by tangkong**
## Description
minor packaging update.  Might also be needed to refresh the conda builds? 🤷 

## Motivation and Context
Time to Tag (TTT)

## How Has This Been Tested?
N/A

## Where Has This Been Documented?
This PR

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive docstrings, including context and API
- [ ] New/changed functions and methods are covered in the test suite where possible
- [ ] Test suite passes locally
- [ ] Test suite passes on GitHub Actions
- [ ] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [ ] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Review by ZLLentz — APPROVED** [review]

**Comment by ZLLentz** [comment]
CI failing on test suite issues with numpy 2.0

File: docs/source/releases.rst [hunk 1]
```diff
@@ -2,6 +2,19 @@ Release History
 ###############
 
 
+v1.5.6 (2024-08-20)
+===================
+
+Maintenance
+-----------
+- Updates test suite and some plans for numpy 2.0 compatibility (primarily with updated bluesky >1.12.0)
```
Comment (ZLLentz) [inline docs/source/releases.rst:10]: I think we shouldn't release a tag that says 2.0 compatibility and then pins <2.0
Maybe let's just not release nabs right now?

Comment (tangkong) [reply docs/source/releases.rst:10]: har you're right.  I thought we needed to re-release nabs for https://github.com/pcdshub/nabs/issues/81 ? or are you thinking of re-making a tag at the same point in the commit history?

Comment (ZLLentz) [reply docs/source/releases.rst:10]: Agh I'm getting pinned by my own comments

Comment (ZLLentz) [reply docs/source/releases.rst:10]: Maybe I'll just do a manual upload of v1.5.5 and we can table this module for a bit?

Comment (ZLLentz) [reply docs/source/releases.rst:10]: Alternatively: switching `round_` to `round` isn't a very big lift, which is what we were erroring on.

Comment (ZLLentz) [reply docs/source/releases.rst:10]: Nevermind, the actual issue is in `dask`

Comment (tangkong) [reply docs/source/releases.rst:10]: I'm ok either way, if manually uploading 1.5.5 isn't hard maybe that's the simpler solution

Comment (ZLLentz) [reply docs/source/releases.rst:10]: I'll go ahead and manually upload 1.5.5 right now as soon as I remind myself how

Comment (ZLLentz) [reply docs/source/releases.rst:10]: I did eventually figure this out via reading the ci helpers code

Comment (ZLLentz) [reply docs/source/releases.rst:10]: Can we modify the notes to not mislead us later? I think that's enough to merge.
```suggestion
- Updates test suite and some plans for numpy 2.0 compatibility (primarily with updated bluesky >1.12.0).
  Note that the dependencies of nabs are still not numpy 2.0 compatible.
```

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**